kong/router/atc.lua

```diff
@@ -382,12 +383,16 @@ function _M:matching(params)
 
   local request_prefix = matched_route.strip_path and matched_path or nil
 
+    local uri_captures = nil
+    if matched_path and captures and pl_tablex.size(captures) > 1 then
+      uri_captures = captures
+    end
```

Comment:
```suggestion
  local uri_captures = nil
  if matched_path and captures and pl_tablex.size(captures) > 1 then
    uri_captures = captures
  end

```

We use 2 spaces identation.